src/components/MapView/GroundstationLayers/index.tsx

```diff
@@ -0,0 +1,83 @@
+import GeoJSON from 'geojson';
+import React from 'react';
+import { get } from 'lodash';
+import { GeoJSONLayer } from 'react-mapbox-gl';
+import * as MapboxGL from 'mapbox-gl';
+import { LayersMap } from '../../../config/types';
+
+import groundstationDataJson from '../../../data/groundstations/longterm_data.json';
+
+declare module 'geojson' {
+  export const version: string;
+  export const defaults: Object;
+  export function parse(
+    data: Object,
+    propreties: Object,
+    callback?: Function,
+  ): Object;
+}
+
+type GroundstationData = {
+  id: string;
+  index: number;
+  lat: number;
+  lon: number;
+}[];
+
+function onClickCircle(evt: any) {
+  // eslint-disable-next-line
+  console.log(
+    get(evt.features[0], 'properties.index'),
+    get(evt.features[0], 'properties.aimagname'),
+    get(evt.features[0], 'properties.sumname'),
+    get(evt.features[0], 'properties.rasterheight'),
+  );
+}
+
+const groundstationDataGeoJSON = GeoJSON.parse(
+  (groundstationDataJson as unknown) as GroundstationData,
```

Comment:
What's the point of this cast? `parse` doesn't know about our `GroundstationData` type, and we're not actually getting any type-safety since we're just making a cast from `unknown` (TypeScript doesn't add anything in that case).